### ISSUE TITLE
BUG 1D filter resolutions introduces energy issue at octaves

### ISSUE THREAD

**Issue by cyrusvahidi** [kymatio/kymatio#984]
While developing JTFS, we observed some artefacts that appeared around octave boundaries. Moreover, we observed that the energy of a signal that should have constant energy across `n1` decays with increasing `j`. 

This holds for `Scattering1D` as well as JTFS and is exemplified by visualization of the S1 of an dirac or chirp:

```
from kymatio.torch import Scattering1D
import librosa 

def plot_spec(Sx, S, sr=22050):
    x_coords = librosa.times_like(Sx, hop_length=S.T)
    y_coords = [psi["xi"]*sr for psi in S.psi1_f]

    plt.figure(figsize=(8, 4))
    librosa.display.specshow(Sx[1:,:].numpy(), sr=sr,
        x_coords=x_coords, x_axis="time",
        y_coords=y_coords, y_axis="cqt_hz",
        cmap="magma")
    plt.xlabel("Time (seconds)")
    plt.ylabel("Frequency (Hz)")
    plt.ylim(0, sr // 2)
    # plt.yticks([100, 200, 400, 800, 1600, 3200, 6400])
    plt.xticks(np.arange(0, 1.2, 0.2))
    plt.minorticks_off()

N = 2**15
kwargs = {'shape': N, 'Q': (12, 1), 'J': 8}
S = Scattering1D(**kwargs, max_order=1)

x = torch.zeros(N)
x[N // 2] = 1

Sx = S(x)

plot_spec(Sx, S, sr=2**15)
```

![image](https://user-images.githubusercontent.com/10761938/211219386-4863763e-5918-4f2f-90a1-41ee13f9efa8.png)

![image](https://user-images.githubusercontent.com/10761938/211219482-fefbed08-fb3c-415e-b6f1-dd436b695dc8.png)

And the spectrum at the time slice centred around the impulse:

![image](https://user-images.githubusercontent.com/10761938/211219419-036ffa6b-2d2d-44a3-9ef7-364745175efd.png)

Which appears to decay by a factor of 2 at octave boundaries ... an energy decay by roughly `2**k1`

This discontinuity at octave boundaries did not cause problems at second-order for `Scattering1D`, but is an issue for second-order of JTFS because we do wavelet conv across octaves. 

The chirp below covers 3 octaves: 

![image](https://user-images.githubusercontent.com/10761938/211219647-1e6d2d56-6d40-4018-897b-4a77d5fa5d34.png)

What results are some energy artifacts exactly at the octave boundaries:

![image](https://user-images.githubusercontent.com/10761938/211219661-af734f3a-256b-4e22-9706-2ed92e8a8153.png)

We identified some relationship between this and `T`. We tried normalizing the wavelet filters to have constant peak. This worked for computing U1, but not S1. 

We realzing that when we create resolutions of `psi` and `phi`, we subsample by taking a mean, but across different signal lengths. This introduces an energy decay correlated with `2**level`.
https://github.com/kymatio/kymatio/blob/954c44990ca2edcd9b4cc12c20321dd20d27cce6/kymatio/scattering1d/filter_bank.py#L382 
https://github.com/kymatio/kymatio/blob/954c44990ca2edcd9b4cc12c20321dd20d27cce6/kymatio/scattering1d/filter_bank.py#L395 

#983 proposes to solve this by changing the `mean` to a `sum` to preserve l1 norm. we get a nice response at `S1` for a dirac:

![image](https://user-images.githubusercontent.com/10761938/211219812-fe1d271c-48ff-4633-8579-e3448d9658f4.png)

![image](https://user-images.githubusercontent.com/10761938/211219917-d12f6865-6998-4637-b992-a22fe180cd33.png) 

The difference between `n1` and `n1_max` is not huge and the decay is smooth rather than these abrupt and steep jumps between octaves.
